media-libs/libavif/libavif-0.8.0.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Library for encoding and decoding .avif files"
+DESCRIPTION="This library aims to be a friendly, portable C implementation of \
+the AV1 Image File Format, as described here: \
+ \
+https://aomediacodec.github.io/av1-avif/ \
+ \
+It is a work-in-progress, but can already encode and decode all AOM supported \
+YUV formats and bit depths (with alpha). \
+ \
+For now, it is recommended that you checkout/use tagged releases instead of \
+just using the master branch. I will regularly create new versions as \
+bugfixes and features are added."
+HOMEPAGE="https://https://github.com/AOMediaCodec/libavif"
+COPYRIGHT="2019 Joe Drago"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/AOMediaCodec/libavif/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="c917f4a27add61b8ad15f328090942d519460b71219f31e60e5fe6aa8cf3dc4e"
+
+ARCHITECTURES="?x86_gcc2 ?x86 !x86_64 ?arm ?riscv64 ?sparc ?m68k"
```

Comment:
You can mark x86_gcc2 as broken as there is no dav1d version that can be build for that architecture:
```diff
- ARCHITECTURES="?x86_gcc2 ?x86 !x86_64 ?arm ?riscv64 ?sparc ?m68k"
+ ARCHITECTURES="!x86_gcc2 ?x86 !x86_64 ?arm ?riscv64 ?sparc ?m68k"
```